paasta_tools/util/deployment.py

```diff
@@ -0,0 +1,158 @@
+import json
```

Comment:
bikeshed, but I see no reason for the deployments module to be in util - this can just be paasta_tools.deployment

Reply:
yeah, i realized this after splitting the thing out. will relocate.